src/nvim/api/ui_events.in.h

```diff
@@ -82,7 +82,7 @@ void grid_clear(Integer grid)
   FUNC_API_SINCE(5) FUNC_API_REMOTE_IMPL;
 void grid_cursor_goto(Integer grid, Integer row, Integer col)
   FUNC_API_SINCE(5) FUNC_API_REMOTE_IMPL FUNC_API_COMPOSITOR_IMPL;
-void grid_line(Integer grid, Integer row, Integer col_start, Array data)
+void grid_line(Integer grid, Integer row, Integer col_start, Array data, Integer flags)
```

Comment:
I'd suggest add `Boolean wrap` instead. "invalid" is not needed as part of the protocol.

Reply:
Thanks, I was wondering whether it made sense to expose the flags field! Changed this to a `Boolean wrap`.